src/jep/InvocationHandler.java

```diff
@@ -53,11 +55,12 @@
      * @exception JepException
      *                if an error occurs
      */
-    public InvocationHandler(long tstate, long ltarget, Jep jep)
+    public InvocationHandler(long tstate, long ltarget, Jep jep, final boolean functionalInterface)
```

Comment:
I think most people would use Proxy.. but maybe people are using it with their own implementations.